compiler/main/driver.cpp

```diff
@@ -548,7 +548,7 @@ static void setupChplHome(const char* argv0) {
       // Set an extra default CHPL_CONFIG directory
       rc = snprintf(CHPL_CONFIG, FILENAME_MAX, "%s/%s/%s",
```

Comment:
If I understand things correctly, I think this part is no longer needed. Might deserve a comment even if you don't want to do it in this PR.